### DISCUSSION TITLE
Refreshing on a page should not result in a redirect back to overview

### DISCUSSION THREAD

**Discussion by WebKenth** [umami-software/umami#3082] [Ideas]
Everytime i go to a website to view it's data and accidentally refresh, it briefly refreshes the page but then instantly redirects back to Dashboard

**Comment by AriDeve** [comment]
Feel like it may have been introduced in the last release.
Was confused about experiencing said issue as well, as I don't remember it doing that before I updated.

**Comment by mikecao** [comment]
Likely an issue with the latest team redirect feature. Will get it fixed.
